AI/modules/signal/models/transformer/train_kaggle.py

```
@@ -0,0 +1,303 @@
+    # 피처 계산
+    print(">> 피처 계산 중...")
+    processed  = []
+    fail_count = 0
+
+    for ticker in raw_df['ticker'].unique():
+        df_t = raw_df[raw_df['ticker'] == ticker].copy()
+        try:
+            df_t = add_technical_indicators(df_t)
+            df_t = add_multi_timeframe_features(df_t)
+            processed.append(df_t)
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)